src/site/content/en/metrics/cls/index.md

```diff
@@ -332,16 +334,23 @@ available in the following tools:
 - [PageSpeed Insights](https://pagespeed.web.dev/)
 - [WebPageTest](https://webpagetest.org/)
 
-### Measure CLS in JavaScript
+### Measure layout shifts in JavaScript
 
-{% BrowserCompat 'api.LayoutShift' %}
+To measure layout shifts in JavaScript, you use the [Layout Instability API](https://github.com/WICG/layout-instability).
+
+The following example shows how to create a [`PerformanceObserver`](https://developer.mozilla.org/docs/Web/API/PerformanceObserver) to log `layout-shift` entries to the console:
+
+```js
+new PerformanceObserver((entryList) => {
+  for (const entry of entryList.getEntries()) {
+    console.log('Layout shift:', entry);
+  }
+}).observe({type: 'layout-shift', buffered: true});
+```
+
+### Measure CLS in JavaScript
 
-To measure CLS in JavaScript, you can use the [Layout Instability
-API](https://github.com/WICG/layout-instability). The following example shows
-how to create a
-[`PerformanceObserver`](https://developer.mozilla.org/docs/Web/API/PerformanceObserver)
-that listens for unexpected `layout-shift` entries, groups them into sessions,
-and logs the maximum session value any time it changes.
+To measure CLS in JavaScript, you need to group these unexpected `layout-shift` entries into sessions, and calculate the maximum session value. The following example shows how to log CLS to the console any time it changes.
```

Comment:
Should we recommend web-vitals.js here instead, similar to the INP doc?

cc @malchata

Reply:
We do further down (after a lot of caveats!) but yes I think there’s an argument that, like INP, the complexity since adding windowing has outgrown the usefulness of having an embedded snippet in the docs and we should recommend web-vitals.js for most users (and perhaps link to the source for more advanced users who DO want to implant themselves).

if we do that then I wonder if it should be super light on details (like INP is) or pretty detailed (like CLS currently is) or somewhere in the middle?

My gut feel is somewhere in the middle. INP feels too light and we’re just asking people to take out word for it because “it’s complicated”. CLS however maybe has the opposite problem and perhaps tries to explain too much (plus we’d need to add extra explanation to replace what is currently in the code we’d be removing, for the rest of that to still make sense).

WDYT?

Reply:
Sounds good in theory but not sure I understand what the change would actually look like. Curious to also get Jeremy's feedback before picking a direction.

Reply:
My take is that if web-vitals.js makes it easier, we should encourage it over homegrown solutions. I'd drop the perf observer code snippet, IMO and recommend what we've already done in-house.

Reply:
And it's gone in https://github.com/GoogleChrome/web.dev/pull/9845/commits/ef2f05dee7f552af4349175dd4005f97b56d216a